tests/unit/test_script_aggregator.py

```diff
@@ -154,3 +156,60 @@ def test_inventory_aggregator(self, data, expected, file_create_fixture):
             inventory.aggregate(tpath, device_type_path="", artifact_info_path="")
             == expected
         )
+
+
+class TestIdentityAggregator:
+    @pytest.fixture(autouse=True)
+    def set_log_level(self, caplog):
+        caplog.set_level(log.INFO)
+
+    @pytest.fixture
+    def file_create_fixture(self, tmpdir):
+        d = tmpdir.mkdir("inventoryaggregator")
+
+        def create_script(data):
+            f = d.join("script")
+            f.write(data)
+            os.chmod(f, stat.S_IRWXU | stat.S_IRWXO | stat.S_IRWXG)
+            return str(f)
+
+        return create_script
+
+    @pytest.mark.parametrize(
+        "data, expected",
+        [
+            (
+                """#! /bin/sh
+        echo mac=c8:5b:76:fb:c8:75
+    """,
```

Comment:
Not that matters much, but I would align this same way as the other script.
```suggestion
                """#! /bin/sh
                echo mac=c8:5b:76:fb:c8:75""",
```